FrxEditor.inc

```diff
@@ -421,17 +421,26 @@ class FrxEditor {
       'default');
   }
 
-
-  public function addParameters($parms_to_add) {
+  /**
+   * Add parameters to report
+   * @param array $parms_to_add, boolean $keyValue
```

Comment:
The parameter declarations should minimally be:
```
 * @param array $params_to_add
 * @param bool $keyValue
```

See https://www.drupal.org/node/1354